docs/contributing.md

```diff
@@ -11,20 +11,36 @@ Thank you for taking an interest in contributing to Chatwoot. Before submitting
 
 ## Pull Request Guidelines
 
-- We use [git-flow](https://nvie.com/posts/a-successful-git-branching-model/) branching model. The base branch is develop. 
+### General Guidelines
 
-- Comment on an issue before you start work on it. This helps to avoid multiple people working on similar issues.
+- We use [git-flow](https://nvie.com/posts/a-successful-git-branching-model/) branching model. The base branch is `develop`
 
-- Please raise the PR against `develop` branch
+- Please raise your PRs against `develop` branch
 
 - It's okay and encouraged to have multiple small commits as you work on the PR - we will squash the commits before merging.
 
-- If adding a new feature:
-  - Please create the branch in the format `feature/<issue-id>-<issue-name>` (eg: `feature/235-contact-panel`)
-  - Add accompanying test case.
-  - Provide a convincing reason to add this feature. Ideally, you should open a suggestion issue first and have it approved before working on it.
-
-- If fixing bug:
-  - If you are resolving a special issue, add `Bug: Fix xxxx` (#xxxx is the issue) in your PR title.
-  - Provide a detailed description of the bug in the PR.
-  - Add appropriate test coverage if applicable.
+### Getting Started 
+
+- Before starting your work, ensure an [issue](https://github.com/chatwoot/chatwoot/issues) exist for it. If not feel free to create one.
+- Add a comment on the issue and wait for the issue to be assigned before you start working on it. 
+  - This helps to avoid multiple people working on similar issues. 
```

Comment:
Please fix the indentation

Reply:
it's intended to be a secondary level list item